integration/admin/admin_integration_test.go

```diff
@@ -566,6 +566,8 @@ type testServer struct {
 	lis    net.Listener
 	server *grpc.Server
 	conn   *grpc.ClientConn
+
+	dbDone func(context.Context)
```

Comment:
nit: Strictly speaking, at this layer, the code has no idea that this is a DB cleanup function - it thinks it's a registry cleanup function.

Reply:
renamed to `cleanup`